app/views/taxons/version_history.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= display_header title: page.title,
+      breadcrumbs: [:taxons, link_to(page.taxon.title, taxon_path(page.taxon.content_id)), 'Version history'] %>
+
+<% page.version_history.each do |version| %>
+  <div class="well well-sm">
+    <p>
+      <%= version.created_at.to_s(:govuk_date) %>
+      <span class="text-muted">(#<%= version.number %>)</span>
+    </p>
+
+    <% if version.note.present? %>
+      <p class="lead">
+        <%= version.note %>
+      </p>
+    <% end %>
+
+    <dl class="dl-horizontal">
+      <% version.changes.each do |attribute, (before, after)| %>
+        <dt>
+          <%= attribute %>
+        </dt>
+        <dd>
+          <code><%= before.inspect %></code> &rarr; <code><%= after.inspect %></code>
```

Comment:
Would this show 'nil' -> 'something' if the taxon is new? If so, maybe this can be more descriptive?

Reply:
Yes, that's correct. I've added a screenshot to the PR description which shows this.

It was something I thought of while I was working on the versioning, but in the end I decided to leave it as is, because I don't believe it's a deal-breaker. I think we should get it out there and to be used before we start rethinking about how to display the data.